pkgs/c_compiler/README.md

```diff
@@ -1,6 +1,13 @@
 [![dart](https://github.com/dart-lang/native/actions/workflows/dart.yaml/badge.svg)](https://github.com/dart-lang/native/actions/workflows/dart.yaml)
-[![pub package](https://img.shields.io/pub/v/c_compiler.svg)](https://pub.dev/packages/c_compiler)
 [![Coverage Status](https://coveralls.io/repos/github/dart-lang/native/badge.svg?branch=main)](https://coveralls.io/github/dart-lang/native?branch=main)
-<!-- [![package publisher](https://img.shields.io/pub/publisher/c_compiler.svg)](https://pub.dev/packages/c_compiler/publisher) -->
+[![pub package](https://img.shields.io/pub/v/c_compiler.svg)](https://pub.dev/packages/c_compiler)
+[![package publisher](https://img.shields.io/pub/publisher/c_compiler.svg)](https://pub.dev/packages/c_compiler/publisher)
 
 A library to invoke the native C compiler installed on the host machine.
+
+This package is experimental. It might undergo breaking changes or never be
```

Comment:
Here's an example of common verbiage we use for packages published under the `labs.dart.dev` publisher:

https://github.com/dart-lang/http/tree/master/pkgs/cronet_http